transaction-status/src/option_serializer.rs

```diff
@@ -0,0 +1,78 @@
+use serde::{ser::Error, Deserialize, Deserializer, Serialize, Serializer};
+
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum OptionSerializer<T: Default> {
+    Some(T),
+    None,
+    Skip,
+}
+
+impl<T: Default> Default for OptionSerializer<T> {
+    fn default() -> Self {
+        Self::None
+    }
+}
```

Comment:
I'd prefer if `OptionSerializer` didn't have a `Default` impl if possible since it's not clear to me whether `None` or `Skip` should be the default.